### PR TITLE
FIRECERT-2198: Modify Accessibility and Closed captions manage test cases to JSV2 format

### PR DESCRIPTION
**Pull request by PreethiMaai**
https://ccp.sys.comcast.net/browse/FIRECERT-2198

### CODE REVIEW

File: cypress/TestCases/FireboltCertification/Accessibility2198.feature [hunk 1]
```diff
@@ -0,0 +1,109 @@
+Feature: Accessibility
+
+    Background: Launch FCA for 'Accessibility'
+        Given the environment has been set up for 'Accessibility' tests
+        And 3rd party 'certification' app is launched
+
+    @Accessibility @coreSDK @sdk @transport
+    Scenario Outline: Accessibility.closedCaptionsSettings - Positive Scenario: <Scenario>
+        When '3rd party app' registers for the 'Firebolt' 'CLOSEDCAPTION_SETTINGS' event
```
Comment (jbigel) [inline cypress/TestCases/FireboltCertification/Accessibility2198.feature:9]: Let's use the establish GC created here: https://ccp.sys.comcast.net/browse/SYSTEST-9780 so we don't have to use `CLOSEDCAPTION_SETTINGS` in each test step

File: cypress/TestCases/FireboltCertification/Accessibility2198.feature [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            | Set textAlignVertical-top            | textAlignVertical  | top                   |
+            | Set textAlignVertical-middle         | textAlignVertical  | middle                |
+            | Set textAlignVertical-bottom         | textAlignVertical  | bottom                |
+            | Set windowColor-white                | windowColor        | white                 |
+            | Set windowOpacity-50                 | windowOpacity      | 50                    |
+            | Set preferredLanguages               | preferredLanguages | spa,eng               |
+
+    @Accessibility @coreSDK @sdk @transport
+    Scenario Outline: Accessibility.closedCaptionsSettings - Positive Scenario: <Scenario> with 'null' params
+        When '3rd party app' registers for the 'Firebolt' 'CLOSEDCAPTION_SETTINGS' event
```
Comment (jbigel) [inline cypress/TestCases/FireboltCertification/Accessibility2198.feature:54]: same as above

File: cypress/TestCases/FireboltCertification/Accessibility2198.feature [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            | backgroundColor   | backgroundColor   |
+            | backgroundOpacity | backgroundOpacity |
+            | textAlign         | textAlign         |
+            | textAlignVertical | textAlignVertical |
+            | windowColor       | windowColor       |
+            | windowOpacity     | windowOpacity     |
+
+    @Accessibility @coreSDK @sdk @transport
+    Scenario Outline: Accessibility.voiceGuidanceSettings - Positive Scenario: <Scenario>
+        When '3rd party app' registers for the 'Firebolt' 'VOICEGUIDANCE_SETTINGS' event
```
Comment (jbigel) [inline cypress/TestCases/FireboltCertification/Accessibility2198.feature:79]: same as above

File: cypress/TestCases/FireboltCertification/Accessibility2198.feature [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            | Scenario              | Method  | Value |
+            | Disable voiceguidance | enabled | false |
+            | Enable voiceguidance  | enabled | true  |
+            | Set speed-1           | speed   | 1     |
+            | Set speed-0.5         | speed   | 0.5   |
+            | Set speed-2           | speed   | 2     |
+
+    @Accessibility @coreSDK @sdk @transport
+    Scenario Outline: Accessibility.audioDescriptionSettings - Positive Scenario: <Scenario>
+        When '3rd party app' registers for the 'Firebolt' 'AUDIODESCRIPTIONS_SETTINGS' event
```
Comment (jbigel) [inline cypress/TestCases/FireboltCertification/Accessibility2198.feature:97]: same as above

**Review by jbigel — CHANGES_REQUESTED** [review]

File: cypress/TestCases/FireboltCertification/Accessibility2198.feature (hunk 1 above)
Comment (PreethiMaai) [reply cypress/TestCases/FireboltCertification/Accessibility2198.feature]: updated with new test step

**Comment by neeradanelxsi** [comment]
Closing PR as changes have been done in PR https://github.com/rdkcentral/firebolt-certification-suite/pull/147/files

**Pull request closed without merging**